pom.xml

```diff
@@ -2772,6 +2772,11 @@
                 <artifactId>mockito-core</artifactId>
                 <version>4.6.0</version>
             </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-inline</artifactId>
+                <version>4.7.0</version>
+            </dependency>
```

Comment:
Should we parameterize both `mockito-core` and `mockito-inline` versions (should be the same also)?